tests/components/mqtt/test_binary_sensor.py

```diff
@@ -123,7 +123,6 @@ async def test_setting_sensor_value_expires_availability_topic(
                     "name": "test",
                     "state_topic": "test-topic",
                     "expire_after": 4,
-                    "force_update": True,
```

Comment:
We do not need `force_update` here as it enforces to  always write the state. Instead we extend the test to make sure that after expiry the state change is processed, even if the same value is received.